mslib/msui/editor.py

```diff
@@ -35,6 +35,7 @@
 from mslib.msui import constants
 from mslib.msui.constants import MSS_CONFIG_PATH
 from mslib.msui.icons import icons
+import os.path
```

Comment:
please use fs (this is pyfilesystem2)
there are plenty examples in the code base.

By this we for example can use a webdav server like nextcloud/owncloud  and are not limited to local files